src/epr/config.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText: © 2024 Brett Smith <xbcsmith@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+from dataclasses import asdict, dataclass, field
+from typing import Any, Dict, List
+
+
+@dataclass
+class Config:
+    """Data class for Config"""
+
+    url: str
+    token: str
+    debug: bool = False
+
+    events: List[Dict[str, Any]] = field(default_factory=list)
+    event_receivers: List[Dict[str, Any]] = field(default_factory=list)
+    event_receiver_groups: List[Dict[str, Any]] = field(default_factory=list)
```

Comment:
I couldn't tell from usage, why are these lists? Seems like they're only ever length 1 based on cli args

Reply:
When using epr-python as a library for a larger program you could easily pass a list of dicts to the the config.

Reply:
Maybe missing something. As a library, the epr client in this repo doesn't use the config - it's only for the cli. Perhaps `Client` should take a config but it probably doesn't need this stuff since you'd just use method params